server/utils/utils.py

```diff
@@ -0,0 +1,34 @@
+import click
+
+
+def find_available_port(host, port):
+    """
+    Helper method to find open port on host. Tries 50 ports incremented from the specified port
+    """
+    import errno
+    import socket
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
this is the snippet of code I may have written for hca/data-store to find an unused tcp port:

```
    import contextlib
    import socket

    with contextlib.closing(socket.socket()) as sock:
        sock.bind(('127.0.0.1', 0))
        return sock.getsockname()[1]
```

it closes the port cleanly with minimal code.

Reply:
good advice, will do